src/main/java/com/onarandombox/MultiverseCore/MultiverseCorePlaceholders.java

```diff
@@ -0,0 +1,117 @@
+package com.onarandombox.MultiverseCore;
+
+import com.onarandombox.MultiverseCore.api.MVWorld;
+import com.onarandombox.MultiverseCore.api.MVWorldManager;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MultiverseCorePlaceholders extends PlaceholderExpansion {
+
+    private final MultiverseCore plugin;
+
+
+    public MultiverseCorePlaceholders(MultiverseCore plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public @NotNull String getIdentifier() {
+        return "multiverse";
```

Comment:
`multiverse-core` may be better, bcu we may have papi extension for our other submodules